<launch>

  <arg name="drone_name" default="drone" />
  <arg name="use_panel" default="true" />

  <group ns="$(arg drone_name)">

    <param name="drone_name" value="$(arg drone_name)"/>

    <node name="targetsim" pkg="aapacs_drones" type="virtual_drone.py" />

    <node if="$(arg use_panel)" name="controlpanel" pkg="aapacs_drones" type="drone_panel.py" />

  </group>

</launch>
